### ISSUE TITLE
Wrong translation in French for ¨Enable chatting online¨.

### ISSUE THREAD

**Issue by PoorPocketsMcNewHold** [supertuxkart/stk-code#4252]
## Description

![Capture d'écran Deepin_dde-desktop_20200321094254](https://user-images.githubusercontent.com/13170204/77222943-63919200-6b58-11ea-8a28-ec7b8d38722f.png)
Notice the presence of two ¨Activer le chat durant les parties en ligne¨. (Activate the chat during online games).
![Capture d'écran Deepin_dde-desktop_20200321094605](https://user-images.githubusercontent.com/13170204/77223001-cbe07380-6b58-11ea-9a7d-a8904df1d902.png)
Here´s the english one, for comparison.

I´m member of the SuperTuxKart transifex group, but i didn´t know if i should report this translation issue there or here.

## Steps to reproduce

1. Go in the options.
2. Select the French language for the game.
3. Go in the General tab.

## Configuration

STK release version: 1.1

STK Source (PPA, distribution package, official binary, etc.): 
Community Arch Linux distribution package : https://www.archlinux.org/packages/community/x86_64/supertuxkart

**Comment by PoorPocketsMcNewHold** [comment]
Reported on the Transifex string as well https://www.transifex.com/supertuxkart/supertuxkart/translate/#fr/supertuxkartpot/184485464?q=text%3A'Enable+chatting+online'.

**Comment by Benau** [comment]
I saw you translated 100% french... so you have fixed it yourself?

**Comment by PoorPocketsMcNewHold** [comment]
> I saw you translated 100% french... so you have fixed it yourself?

I'm not a Transifex reviewer, while the previous iteration was already accepted, I've opened an issue there, and suggested another text myself.
A reviewer did accept the newly suggested text, but didn't resolve the issue that I've opened. I will close both myself manually then.
And yeah, I've also completed the remaining untranslated strings on Transifex, because why not?
